willow/registry.py

```diff
@@ -76,6 +97,15 @@ def register_plugin(self, plugin):
             self.register_converter(converter[0], converter[1], converter[2])
 
     def get_operation(self, image_class, operation_name):
+        if operation_name in self._registered_generic_operations:
+            generic_operation = self._registered_generic_operations[operation_name]
+
+            for dependency in generic_operation['dependencies']:
+                if dependency not in self._registered_operations[image_class]:
+                    raise LookupError("Generic operation dependencies not met")
```

Comment:
It would be helpful to include which dependency (if not dependencies) are not met

Reply:
This error doesn't actually get seen by end user's it'll be swallowed up by the ``find_operation`` method